content/chainguard/libraries/faq.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Chainguard Libraries FAQ"
+linktitle: "FAQ"
+type: "article"
+description: "Frequently Asked Questions and answers for Chainguard Libraries users"
+draft: false
+menu:
+  docs:
+    parent: "libraries"
+weight: 003
+toc: true
+---
+
+
+## Frequently Asked Questions
```

Comment:
We don't need this header as the title header will be there

Reply:
Removed.